<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dipendente</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_nomecognome">
      <property name="font">
       <font>
        <pointsize>38</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_id">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_cf">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Codice Fiscale: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_indirizzo">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Indirizzo: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_email">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>E-mail:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_telefono">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Telefono: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_datanascita">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Data di nascita: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_luogonascita">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Luogo di nascita: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_licenza">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Licenza: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
